assets/manifest.json

```diff
@@ -41,6 +41,7 @@
   "permissions": [
     "https://github.com/",
     "https://gist.github.com/",
-    "https://githublinker.herokuapp.com/"
+    "https://githublinker.herokuapp.com/",
+    "storage"
```

Comment:
We should declare `storage` as an optional permission. Otherwise the update have to be confirmed by the user, because of the new permissions.